bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -0,0 +1,234 @@
+import asyncio
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Roles
+
+
+logger = logging.getLogger(__name__)
+
+
+ANNOYED_EXPRESSIONS = ["-_-", "-.-"]
+
+WRONG_ANS_RESPONSE = [
+    "No one gave the correct answer",
+    "Better luck next time"
+]
+
+
+class TriviaQuiz(commands.Cog):
+    """A cog for all quiz commands."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.questions = self.load_questions()
+        self.game_status = {}
+        self.game_owners = {}
+        self.question_limit = 4
+        self.player_dict = {}
+        self.categories = {
+            "general": "Test your general knowledge"
+            # "retro": "Questions related to retro gaming."
+        }
+
+    @staticmethod
+    def load_questions() -> dict:
+        """Load the questions from  json file."""
+        p = Path("bot", "resources", "evergreen", "trivia_quiz.json")
+        with p.open() as json_data:
+            questions = json.load(json_data)
+            return questions
+
+    @commands.command(name="quiz", aliases=["trivia", "tquiz"])
```

Comment:
```suggestion
    @commands.command(name="quiz", aliases=["trivia"])
```
I can't imagine anyone trying out `tquiz` on their first go unless they explicitly knew about it in advance, and even then, `quiz` is shorter and easier to remember.